Problem: we don't have enough packages

yrashk opened the pull request:
Solution: import the bulk of packages from Pigsty

For now, we'll select all packages that are on github or gitlab. Some of them will build, some won't. But we'll know better once we try.

jonatas commented on pigsty.rb:
```diff
@@ -0,0 +1,36 @@
+    origin = row["url"]
+    if row["url"] =~ /github/
+      github = origin.match(%r{https://github\.com/([^/]+)/([^/]+)}).captures.join("/")
+      origin = "github \"#{github}\""
```
Idea: group by github workspace to also detect workspaces for cases like omnigres/timescale.

yrashk replied:
🤔 Maybe we can do this on a case-by-case scenario. Like: should we do this for personal github accounts? What should we do for non-github projects?

jonatas reviewed (approved):
This looks so smart!

We may also think about having pigsty as a "source provider" and create a concept of dynamic packages from source providers, but in this way we can import and start fixing case by case looks much better for long term.

yrashk commented:
The ultimate dream is that this will help @Vonng bring in and maintain packages for Pigsty's distributions, eliminating the tedious work. 

However, as a backup, we can look into "dynamic package sources" where feasible, where we can translate a formal build definition into our format. I like this idea.

yrashk committed:
Problem: we don't have enough packages

Solution: import the bulk of packages from Pigsty

For now, we'll select all packages that are on github or gitlab. Some of
them will build, some won't. But we'll know better once we try.

yrashk committed:
Problem: duplicate packages potentially imported from Pigsty

Solution: ensure we check github urls to avoid duplicates
